detection-rules/brand_impersonation_godaddy

```diff
@@ -0,0 +1,34 @@
+name: "Brand Impersonation: GoDaddy"
+description: "Detects messages where the sender is impersonating GoDaddy through display name manipulation or lookalike domains, while not being legitimately authenticated from GoDaddy's infrastructure."
+type: "rule"
+severity: "medium"
+source: |
+  type.inbound
+  and (
+    regex.icontains(sender.display_name, 'godaddy')
+    or strings.ilike(sender.display_name, "*godaddy*")
+    or strings.ilevenshtein(sender.display_name, 'godaddy') <= 1
+    or strings.ilike(sender.email.domain.domain, '*godaddy*')
+  )
+  and not (
+    sender.email.domain.root_domain == "godaddy.com"
+    and any(headers.hops, .authentication_results.dmarc == "pass")
```

Comment:
might want `and headers.auth_summary.dmarc.pass` instead (doing that from memory, so please double check